1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+    ```
+
+1.  Launch a TTY process inside of the container with the task ID (`<task_id>`) specified. This will launch an interactive Bash session.
+
+    ```bash
+    $ dcos task exec --interactive --tty <task_id> bash
+    ```
+    
+    You should now be inside the container running an interactive Bash session.
+    
+    ```bash
+    root@ip-10-0-2-53 / #
+    ```
+    
+    **Tip:** You can use shorthand abbreviations `-i` for `--interactive` or `-t` for `--tty`. Also, only the beginning unique characters of the `<task_id>` are required. For example, if your task is named `exec-test_20161214195` and there are no other task IDs that begin with the letter `e`, this is valid command syntax: `dcos task exec -i -t e bash`. For more information, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+    
+1.  Run a command from the interactive Bash session from inside the cluster. For example, the `ls` command:
+
+    ```bash
+    root@ip-10-0-1-104 / # ls
+    bin   dev  home  lib64	     media  opt   root	sbin  sys  usr
+    boot  etc  lib	 lost+found  mnt    proc  run	srv   tmp  var
+    ```
+
+# Pipe output from a command running inside a container 
+
+You can run commands inside a container by using the `dcos exec` command. In this example, a long running Marathon app is launched and then the `dcos task exec` command is used to get the hostname of the node running your app. 
+
+1.  Create a Marathon app definition and name it `my-app.json` with the following contents:
+
+    ```bash
+    {
+       "id": "/my-app",
+       "cmd": "sleep 100000000",
+       "instances": 1,
+       "cpus": 1,
+       "mem": 128,
+       "disk": 0,
+       "gpus": 0,
+       "backoffSeconds": 1,
+       "backoffFactor": 1.15,
+       "maxLaunchDelaySeconds": 3600,
+       "upgradeStrategy": {
+         "minimumHealthCapacity": 1,
+         "maximumOverCapacity": 1
+       },
+       "portDefinitions": [
+         {
+           "protocol": "tcp",
+           "port": 10000
+         }
+       ],
+       "requirePorts": false
+     }
+     ```
+
+1.  Deploy the service on DC/OS:
+
+    ```bash
+    $ dcos marathon app add my-app.json
+    ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME        HOST        USER  STATE  ID                                               
+    my-app  10.0.1.106  root    R    <task_id>
+    ```
+
+1.  Run this command to show the hostname of the container running your app, where `<task-ID>` is your task ID.
+
+    ```bash
+    $ dcos task exec <task_id> hostname
+    ```
+    
+    The output should look similar to this:
+    
+    ```bash
+    ip-10-0-1-105.us-west-2.compute.internal
+    ```
+
+For more information about the `dcos task exec` command, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+
+# Run an interactive command on a remote machine
```

Comment:
It's not about running an interactive command on a remote machine. It's about running an interactive command inside your task's container (regardless of the machine it's on)